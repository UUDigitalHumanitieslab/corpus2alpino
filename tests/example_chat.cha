@UTF8
@Begin
@Languages:	nld
@Participants:	CHI Target_Child, INV Investigator
@ID:	nld|PRELAN|CHI|11;00.|male|premature||Target_Child|||
@ID:	nld|test_corpus|INV|||||Investigator|||
@Media:	example, audio
*CHI:	dus pakte zij gereedschap .
*CHI:	toen <ging zij &-eh haar &+ma ging zij> [//] wou zij de auto
	maken . 21792_23475
%xsid:	42
*CHI:	maar toen reed de auto er vandoor .
*CHI:	toen <kwam de> [//] deden ze een wedstrijdje met een andere bus .
@End
