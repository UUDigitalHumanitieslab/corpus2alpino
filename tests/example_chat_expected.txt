##META text charencoding = UTF8
##META text childage = 11;00.
##META int childmonths = 132
##META text media = example, audio
##META text session = example_chat

##META text SES = 
##META text age = 11;00.
##META text corpus = PRELAN
##META text custom = 
##META text education = 
##META text group = premature
##META text language = nld
##META int months = 132
##META text origutt = dus pakte zij gereedschap .
##META text parsefile = PRELAN_example_chat_u00000000001.xml
##META text role = Target_Child
##META text sex = male
##META text speaker = CHI
##META int uttendlineno = 8
##META int uttid = 1
##META int uttno = 1
##META int uttstartlineno = 8
1|dus pakte zij gereedschap .

##META text origutt = toen <ging zij &-eh haar &+ma ging zij> [//] wou zij de auto maken . 21792_23475
##META text parsefile = PRELAN_example_chat_u00000000002.xml
##META int uttendlineno = 10
##META int uttid = 42
##META int uttno = 2
##META int uttstartlineno = 9
##META text xsid = 42
42|toen wou zij de auto maken .

##META text origutt = maar toen reed de auto er vandoor .
##META text parsefile = PRELAN_example_chat_u00000000003.xml
##META int uttendlineno = 12
##META int uttid = 3
##META int uttno = 3
##META int uttstartlineno = 12
##META text xsid = 
3|maar toen reed de auto er vandoor .

##META text origutt = toen <kwam de> [//] deden ze een wedstrijdje met een andere bus .
##META text parsefile = PRELAN_example_chat_u00000000004.xml
##META int uttendlineno = 13
##META int uttid = 4
##META int uttno = 4
##META int uttstartlineno = 13
4|toen deden ze een wedstrijdje met een andere bus .

